feeds/wifi-trunk/hostapd/patches/908-ubus-fix-client-events.patch

```diff
@@ -0,0 +1,111 @@
+--- a/src/ap/ubus.c
++++ b/src/ap/ubus.c
+@@ -578,6 +578,7 @@ static int hostapd_clear_session(struct
+ 	{
+ 		if (session_id == rec->session_id) {
+ 			/* dec counter and delete */
++			wpa_printf(MSG_INFO, "%s Deleting client session with session id %llu", __func__, session_id);
```

Comment:
This can be DEBUG print I guess, many client going in and out will cause lot of prints.

Reply:
There is no easy way to turn hostapd debug at run time. It is an important log which tells if we were successfully able to clear session from hostapd after being processed by SM. I would like to keep it info atleast till the time we have resolved  other client events issues on the cloud side and have done testing with a greater number of clients. May be we can make it debug later?